apcd-cms/docker-compose.dev.yml

```diff
@@ -5,8 +5,8 @@ services:
     ports:
       - 8000:8000
     command: ["python3", "manage.py", "runserver", "0.0.0.0:8000"]
-    container_name: core_cms
-    hostname: core_cms
+    container_name: demdata_cms
+    hostname: demdata_cms
```

Comment:
The [latest build failed](https://jenkins01.tacc.utexas.edu/job/Core_Portal_Deploy/2102/console) because of this prefix change:

```
django.db.utils.OperationalError: could not translate host name "core_cms_postgres" to address: Temporary failure in name resolution
```

1. Is a fix to change the CMS secrets on the server to point to `demdata_cms_postgres`? _I can stop caring._
2. Should we just go back to `core_cms_` and solve the local dev conflict later? _I vote yes._